homeassistant/components/remote_calendar/strings.json

```diff
@@ -18,14 +21,14 @@
       "already_configured": "[%key:common::config_flow::abort::already_configured_service%]"
     },
     "error": {
-      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "cannot_connect": "[%key:component::remote_calendar::common::unable_to_fetch%]",
```

Comment:
I kind of liked the old friendly/standard error. Can we make it use `timeout` for timeout errors?

Reply:
or the existing `timeout_connect` is probably fine while "Timeout establishing connection" may be technically not necessarily accurate, it seems to be mostly correct.

Reply:
Okay